src/api/namespaces/schema.ts

```diff
@@ -15,6 +16,29 @@ export const NamespaceCreatedSchema = z.object({
   namespace: NamespaceSchema,
 });
 
+export const NamespaceLogSchema = z.object({
```

Comment:
I forgot to type comment (I just did that, it can be e.g "instance" 78b9b0c6a93ba03ab4e5982f336ffd9ae9875449). 

The thing is, we don't even need anything within the tags atm, we could even delete this validation node and it would have no effect.

But this brings up an interesting question about how we deal with fields that we don't need right now. Should we not include them in the schema at all (and risk possible future validation issues for fields that we don't even use) or should we keep them as part of our "documentation". I mean they could become handy when we quickly introduce a new feature with a field that we never used before. 


---
Maybe @h8718 can help us out here? It's about the result of a namespace log (like `/api/namespaces/NAMESPACE/logs`)

- what does "source" refer to? It is something like `c75454f2-3790-4f36-a1a2-22ca8a4f8020`
- "callpath" seems to be redundant with "instanceId" any idea what the point of it is?

Reply:
> Should we not include them in the schema at all (and risk possible future validation issues for fields that we don't even use) or should we keep them as part of our "documentation".

Yes, good point, I am not sure about that myself. In a way it would be more efficient and straightforward to avoid definitions we never use. In some way they are "dead code", except they add dependencies that could break when the API changes. On the other hand, I do like this "documentation layer".

I think I have a slight preference for not validating props that we don't use, but I'm fine either way.

If we decide only to declare what we really use, we could add comments on top of the schema files with examples of back-end responses [like I did here](https://github.com/direktiv/direktiv-ui/blob/84afca459821f7307e2d9d14ad37c4584efa4f50/src/api/eventListeners/schema.ts). What do you think about that?

Reply:
`source` is the `source` where the log message originated from. Since at the moment the backend has a api-point for each `source-type` you don't need to care about that it does not have to be in the schema. But if its in its also not a problem since this value will be present anyway even if we completely redo the logging backend.  

We use it internally to filter the logs and return only relevant log messages.

Reply:
Thanks for clarifiying @h8718!


As just discussed, I removed the unused attribute from our schema and added an example response as a comment (becf6637473f6607365a35a5104b910fd19a77a6)